src/config/chis-civ/config.json

```diff
@@ -0,0 +1,199 @@
+      "hierarchy": [
+        {
+          "friendly_name": "Disctrict",
+          "property_name": "district",
+          "contact_type": "c30_district",
+          "type": "name",
+          "required": true,
+          "level": 1
+        }
+      ],
+      "replacement_property": {
+        "friendly_name": "Affected Supervision Area",
+        "property_name": "replacement",
+        "type": "string",
+        "parameter": ["\\(*Supervision*\\)"],
+        "required": true
+      },
+      "place_properties": [
+        {
+          "friendly_name": "Supervision Area Name",
+          "property_name": "name",
+          "type": "generated",
+          "parameter": "Zone de supervision de: {{ contact.last_name }} {{ contact.first_name }}",
+          "required": false
+        }
+      ],
+      "contact_properties": [
+        {
+          "friendly_name": "Last Name",
+          "property_name": "last_name",
+          "type": "string",
+          "parameter": ["\\(*Last*\\)"],
+          "required": true
+        },
+        {
+          "friendly_name": "First Name",
+          "property_name": "first_name",
+          "type": "string",
+          "parameter": ["\\(*First*\\)"],
+          "required": true
+        },
+        {
+          "friendly_name": "Full Name",
+          "property_name": "name",
+          "type": "generated",
+          "parameter": "{{ contact.last_name }} {{ contact.first_name }}",
+          "required": false
+        },
+        {
+          "friendly_name": "Function",
+          "property_name": "function",
+          "type": "string",
+          "required": false
+        },
+        {
+          "friendly_name": "Gender",
+          "property_name": "sex",
+          "type": "select_one",
+          "parameter": {
+            "male": "Masculin",
+            "female": "Feminin"
+          },
+          "required": true
+        },
+        {
+          "friendly_name": "Telephone",
+          "property_name": "phone",
+          "type": "regex",
+          "parameter": "^\\d{10}$",
+          "required": true
+        }
+      ]
+    },
+    {
+      "name": "c70_chw_site",
+      "friendly": "CHW Site",
+      "contact_type": "c72_chw_site",
+      "user_role": ["chw_min_5km", "chw_max_5km"],
+      "username_from_place": true,
+      "deactivate_users_on_replace": true,
+      "hierarchy": [
+        {
+          "friendly_name": "Locality",
+          "property_name": "locality",
+          "contact_type": "c60_locality",
+          "type": "name",
+          "required": true,
+          "level": 1
+        }
+      ],
+      "replacement_property": {
+        "friendly_name": "Affected CHW Site",
+        "property_name": "replacement",
+        "type": "string",
+        "parameter": ["\\(*Site*\\)"],
+        "required": true
+      },
+      "place_properties": [
+        {
+          "friendly_name": "CHW Site Name",
+          "property_name": "name",
```

Comment:
can the CHW site name be generated? 
if so, then username_from_place can also be false